.github/workflows/pr_verification.yml

```
@@ -0,0 +1,38 @@
+---
+name: PR verification workflow
+
+on:
+  pull_request:
+  workflow_dispatch:
+
+jobs:
+  checkstyle:
+    runs-on: ubuntu-latest
+    container:
+      image: fedora:37
+    steps:
+      - name: Check out repository code
+        uses: actions/checkout@v3
+      - name: Install dependencies
+        shell: bash
+        run: sudo dnf install -y gettext jss
```

- `sudo` is not needed here, you are already "root" in the container
- pass also `--setopt install_weak_deps=False` to `dnf` to avoid installing weak dependencies (ie recommends) by default; right now it won't make a difference, however it is more future-proof
```suggestion
        run: dnf --setopt install_weak_deps=False install -y gettext jss
```

---

Thanks. Done.